### ISSUE TITLE
Issues when loading one episode

### ISSUE THREAD

**Issue by yanii** [carla-simulator/imitation-learning#5]
First, thanks for releasing your dataset!

I'm trying to load it using h5py, and I've found that one file in the training set (`data_06790.h5`) consistently causes an error of the sort:

`IOError: Unable to open file (Unable to find a valid file signature)`

Removing this episode from the dataset, everything else loads. I've downloaded the data twice now, and both downloads have the same MD5 hash, so I don't think it's corruption from the download:

`8b61e53ec55e1f79004d79fb61ab8d6c  CORL2017ImitationLearningData.tar.gz`

**Comment by felipecode** [comment]
Hello @yanii 
I think this is probably correct. It could happen that some of parts of the data were corrupted on the way.
My data loading system would just ignore corrupted h5 files, so I probably missed this one.

**Comment by yanii** [comment]
Oh, OK, so it's likely that the results you reported also excluded that file? Hoping to have some comparable numbers on the metric front :-)

Also.. I'm not 100% sure yet, but when loading the data vector (a.k.a. "targets") as a list, I seem to get a different ordering of data to the ordering of the labels you list in the documentation here: https://github.com/carla-simulator/imitation-learning

Could you confirm the ordering I should expect?

**Comment by felipecode** [comment]
Yes, the numbers I reported likely also excluded this file. But anyway i think 200 images shouldn't make much difference on the model :-) .

I will download again and check. Maybe there is some mistake since I was not using all the variables. 
Thanks again for contributing !

**Comment by yanii** [comment]
The target data vector is in the correct order by the way, false alarm on that front, I had messed up something!

**Comment by Fengmoon93** [comment]
import h5py
filename='data_06790.h5'
try:
    data = h5py.File(filename, 'r')
except:
    print('file has bugs') #the output is 'file has bugs'
